src/services/exile_service.py

```diff
@@ -1,34 +1,81 @@
 import discord
 import logging
-from util import log_info_and_embed, add_and_remove_role, send_dm
-from enums import Role
+from util import log_info_and_embed, add_and_remove_role, send_dm, user_has_role
+from enums import Role, ExileStatus
+from database import users_database, exiles_database
+from typing import Optional
+import datetime
+from database.models import Exile, User
 
 logger = logging.getLogger(__name__)
 
 
 async def exile_user(
-    logging_embed: discord.Embed, user: discord.Member, duration: str, reason: str
+    logging_embed: discord.Embed,
+    user: discord.Member,
+    duration: Optional[datetime.timedelta],
+    reason: str,
 ):
-    # TODO implement this functionality
-    log_info_and_embed(
-        logging_embed, logger, f"going to exile {user.mention} for {duration}"
+    # change user role
+    await add_and_remove_role(
+        user, role_to_add=Role.EXILED, role_to_remove=Role.VERIFIED
     )
 
-    # Look up 
-
-    log_info_and_embed(logging_embed, logger, f"because {reason}")
+    # message user
+    await send_dm(
+        user, f"You are being exiled from NAUR FFXIV for the following reason: {reason}"
+    )
```

Comment:
i think we should move this near the end after all the db interactions are completed

Reply:
I think I had just misread a request that Claire had about unexile, I agree we should move this to then end, updated

Reply:
i think we deleted this line on accident

Reply:
That's just a merge conflict thing, there was two calls to each of those and now they exist at the bottom of the method